libs/elements/layout/src/lib/components/banners/elewa-group-image-and-text-banner/elewa-group-image-and-text-banner.component.html

```diff
@@ -0,0 +1,36 @@
+<div
+  class="banner-container"
+  *ngIf="placement !== 'right'"
+  [ngStyle]="{ 'background-color': background }"
+>
```

Comment:
use *ngIf="placement == 'right'; else  'imageLeft'"

and put and #imageLeft tag on an ng-template